app/CommandExecutor.php

```diff
@@ -13,6 +13,21 @@ class CommandExecutor
     /** @var array $commands */
     protected $commands = [];
 
+    /** @var string $outputFileName */
+    protected $outputFileName = '';
```

Comment:
what if we default this do `/dev/null`, `file_get_contents` and `unlink` should work just fine, and simplify checks bellow, just check if file were created if not, don't change this and just use `/dev/null`